src/bundle/Resources/public/js/scripts/fieldType/base/base-field.js

```diff
@@ -40,14 +85,43 @@
             return errorNode;
         }
 
+        /**
+         * Find the error containers
```

Comment:
Would it be a good idea to point out methods that should be overriden when extending a class?